realm/realm-library/src/main/java/io/realm/internal/UncheckedRow.java

```diff
@@ -188,6 +188,13 @@ public LinkView getLinkList(long columnIndex) {
         return new LinkView(context, parent, columnIndex, nativeLinkViewPtr);
     }
 
+    @Override
+    public void checkIfBacklinkAvailable() {
```

Comment:
This function has the wrong name.  It is not checking to see if backlinks are available.  It is checking to see if the object is attached.